RoboFile.php

```diff
@@ -594,10 +594,17 @@ public function writeReleaseVersion($version) {
       ->run();
   }
 
-  public function jiraReleaseVersion($version = null, $opts = []) {
-    if ($version) {
-      $this->validateVersion($version);
+  public function nextReleaseVersion($version = null) {
+    if (!$version) {
+      $version = $this->getReleaseVersionController()
+        ->determineNextVersion();
     }
+    $this->validateVersion($version);
+    return $version;
```

Comment:
This is also a command, right? Wouldn't returning a version cause it be "failed" (return code != 0)?

Reply:
This command is not intended for CI use, more for local debugging. Anyway, its returned value is converted to an exit code of 0. Must be exactly an integer between 0-255 to serve as an error code.